LSS Master

ymkim92 opened the pull request:
Hi Chris,

I added LSS Master. I hope you like it.
This is the test code:

```python
import canopen
import logging

logging.basicConfig(level=logging.DEBUG)

network = canopen.Network()
network.connect(channel='PCAN_USBBUS1', bustype='pcan', bitrate=250000)
#network.connect(channel='PCAN_USBBUS1', bustype='pcan', bitrate=500000)

network.lss.send_switch_mode_global(network.lss.CONFIGURATION_MODE)
node_id = network.lss.inquire_node_id()
print node_id
network.lss.configure_node_id(node_id+1)
network.lss.configure_bit_timing(3)
#network.lss.store_configuration()
#network.lss.send_switch_mode_global(network.lss.NORMAL_MODE)

network.disconnect()
```

Regards,
Young

christiansandberg commented on canopen/lss.py:
```diff
@@ -0,0 +1,160 @@
+        
+        message = [0]*8
+
+        if len(message) != 8:
```
This should not be necessary since you define it as 8 items long.

ymkim92 commented:
Hi christiansandberg,

I made fixes as you requested:
- changed the use of Listener interface to Network.subscribe
- add lss.rst
- removed unnecessary length check
 
Regards,
Young
